app/src/main/java/com/courseproject/tindar/ui/home/SecondViewProfileFragment.java

```diff
@@ -184,33 +189,26 @@ public void onDestroyView() {
     /**
      * Downloads image from link.
      */
-    private class DownloadImage extends AsyncTask<String, Void, Bitmap> {
-        ImageView profilePicture;
-
-        public DownloadImage(ImageView profilePicture) {
-            this.profilePicture = profilePicture;
-        }
-
-        protected Bitmap doInBackground(String... urls) {
-            String urldisplay = urls[0];
-            Bitmap mIcon11 = null;
-            try {
-                InputStream in = new java.net.URL(urldisplay).openStream();
-                mIcon11 = BitmapFactory.decodeStream(in);
-            } catch (Exception e) {
-            }
-            return mIcon11;
-        }
-
-        protected void onPostExecute(Bitmap result) {
-            profilePicture.setImageBitmap(result);
-        }
-    }
 
     private void setsProfile() {
         ViewProfileResponseModel profile =
                 viewProfilesController.getProfile(otherUserId);
 
+        profileLink = profile.getProfilePictureLink();
+        if (profileLink.equals("spongebob1.png")){
```

Comment:
can you comment on it right above this line 
// This is only a temporary implementation for demo purposes. The plan is to connect to show the picture from the external URL.